### PR TITLE
Move prefect flows from secure to job-dsl

### PR DESCRIPTION
**Pull request by AbdouSeck**
The PR moves the list of prefect flows from secure to here. [Here is the JIRA ticket](https://openedx.atlassian.net/browse/DENG-1048) for some context.

The main code change is the addition of the names of the flows to the updated file as a `List` of strings. Subsequently, the list is processed using `List.each {}`.

Please let me know if you have any questions.

### CODE REVIEW

**Comment by edx-webhook** [comment]
Thanks for the pull request, @AbdouSeck! I've created [OSPR-6461](https://openedx.atlassian.net/browse/OSPR-6461) to keep track of it in JIRA, where we prioritize reviews. Please note that it may take us up to several weeks or months to complete a review and merge your PR.

Feel free to add as much of the following information to the ticket as you can:
- supporting documentation
- [Open edX discussion forum threads](https://discuss.openedx.org/)
- timeline information ("this must be merged by XX date", and why that is)
- partner information ("this is a course on edx.org")
- any other information that can help Product understand the context for the PR

All technical communication about the code itself will be done via the GitHub pull request interface. As a reminder, [our process documentation is here](http://edx-developer-guide.readthedocs.org/en/latest/process/overview.html).


Please let us know once your PR is ready for our review and all tests are green.


:warning: We can't start reviewing your pull request until you've submitted a     [signed contributor agreement](https://openedx.org/cla)     or indicated your institutional affiliation. Please see the [CONTRIBUTING](https://github.com/edx/edx-platform/blob/master/CONTRIBUTING.rst)     file for more information.     If you've signed an agreement in the past, you may need to re-sign.  See     [The New Home of the Open edX Codebase](https://open.edx.org/blog/the-new-home-of-the-open-edx-codebase/)     for details.

File: dataeng/jobs/analytics/PrefectFlowsDeployment.groovy [hunk 1]
```diff
@@ -94,7 +94,23 @@ class PrefectFlowsDeployment{
...
+            'load_enterprise_tables_from_s3_to_aurora',
+            'load_geoip_to_snowflake',
+            'load_google_analytics_data_to_snowflake',
+            'load_google_sheets_to_snowflake',
+            'load_paypal_cases_to_snowflake',
+            'load_paypal_settlement_report_to_snowflake',
+            'load_paypal_transaction_detail_report_to_snowflake',
+            'send_hubspot_leads_to_braze',
+            'send_hubspot_unsubscribes_to_braze',
+            'load_segment_config_to_snowflake'
```
Comment (pwnage101) [inline dataeng/jobs/analytics/PrefectFlowsDeployment.groovy:111]: Nit: use trailing commas wherever possible.

**Review by pwnage101 — APPROVED** [review]

**Comment by natabene** [comment]
@AbdouSeck If you rebase, this should clear the CLA check.

**Comment by AbdouSeck** [comment]
Thanks @natabene! That did it.

**Pull request merged**